.github/workflows/ci-packages.yaml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
       - 'packages/**'
       - '.github/workflows/ci-packages.yaml'
+      - '.github/actions/ci/action.yaml'
```

Comment:
The changes in `.github/actions/build/action.yaml` should be covered by `.github/workflows/ci-examples.yaml`.